### PR TITLE
User can now view static images below the carousel to immediately browse to category.

### PR DESCRIPTION
**Pull request by aaronkeith**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -30,9 +30,58 @@ body {
   color: #83fe00;
   text-decoration: none;
 }
+
+#static-caption {
```
Comment (thebearingedge) [inline styles.css]: You need to commit to a casing convention. You should use all `kebab-case` for your ids. some of them are `camelCase`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Good work, just a small change.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Once you fix the merge conflict, go ahead and merge this yourself.

**Comment by aaronkeith** [comment]
resolved merge conflicts

**Pull request merged**